### DISCUSSION TITLE
Where is the option to zoom in further?

### DISCUSSION THREAD

**Discussion by zediaa** [arichornlover/uYouEnhanced#246] [Q&A]
On previous versions of uyou (i havent always used enhanced), when I zoomed in  it would zoom it as far as I wanted. Now, it only goes to fill screen. I have looked a few different times to find an option I may have turned off. Does anyone know where it is if it exists? Otherwise, thanks for updating the app to remove shorts! Works great.

**Comment by ego-lay-atman-bay** [comment]
The remove modern ui option disables zooming (or at least it did last time I enabled it), so if you have it on, that may be why you can't zoom.
